SonarQube.Bootstrapper/BuildAgentUpdater.cs

```diff
@@ -47,7 +44,7 @@ public bool TryUpdate(string hostUrl, string targetDir, ILogger logger)
             {
                 try
                 {
-                    logger.LogMessage(Resources.INFO_Downloading, BootstrapperSettings.SonarQubeIntegrationFilename, integrationUrl, downloadedZipFilePath);
+                    logger.LogDebug(Resources.INFO_Downloading, BootstrapperSettings.SonarQubeIntegrationFilename, integrationUrl, downloadedZipFilePath);
```

Comment:
Downloading is fairly important and potentially slow step so I think the standard output should give some indication that it is happening (whether it's this message or one generated by something higher up the stack).